db-tests/src/test/java/gov/nasa/jpl/aerie/database/ActivityDirectiveChangelogTests.java

```diff
@@ -0,0 +1,310 @@
+package gov.nasa.jpl.aerie.database;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+import java.io.File;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.UUID;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class ActivityDirectiveChangelogTests {
+
+  private static final File initSqlScriptFile = new File("../merlin-server/sql/merlin/init.sql");
+  private DatabaseTestHelper helper;
+  private MerlinDatabaseTestHelper merlinHelper;
+
+  private Connection connection;
+  int fileId;
+  int missionModelId;
+
+  @BeforeEach
+  void beforeEach() throws SQLException {
+    fileId = merlinHelper.insertFileUpload();
+    missionModelId = merlinHelper.insertMissionModel(fileId);
+  }
+
+  @AfterEach
+  void afterEach() throws SQLException {
+    helper.clearTable("uploaded_file");
+    helper.clearTable("mission_model");
+    helper.clearTable("plan");
+    helper.clearTable("activity_directive_changelog");
+    helper.clearTable("activity_directive");
+  }
+
+  @BeforeAll
+  void beforeAll() throws SQLException, IOException, InterruptedException {
+    helper = new DatabaseTestHelper(
+        "aerie_merlin_test",
+        "Merlin Database Tests",
+        initSqlScriptFile
+    );
+    helper.startDatabase();
+    connection = helper.connection();
+    merlinHelper = new MerlinDatabaseTestHelper(connection);
+  }
+
+  @AfterAll
+  void afterAll() throws SQLException, IOException, InterruptedException {
+    helper.stopDatabase();
+    connection = null;
+    helper = null;
+  }
+
+  @Test
+  void shouldHaveNoChangelogsForEmptyPlan() throws SQLException {
+    final var expectedCount = 0;
```

Comment:
Since this variable is only used once, you can just say `assertEquals(0, count);` The name of the test/syntax of `assertEquals` implies that `0` is the "expected value of count". Additionally, by having `0` inlined here, the test is easier to understand, as you don't have to look away from the assert statement to know what we're asserting `count` is equal to.

In general, you don't need to declare these "expected" variables if they're primitives or primative-like (ie Strings, Durations, ints, longs, booleans, etc).